polkadot-parachain/src/chain_spec/asset_hubs.rs

```diff
@@ -67,35 +67,32 @@ pub fn asset_hub_polkadot_development_config() -> AssetHubPolkadotChainSpec {
 	properties.insert("tokenSymbol".into(), "DOT".into());
 	properties.insert("tokenDecimals".into(), 10.into());
 
-	AssetHubPolkadotChainSpec::from_genesis(
-		// Name
-		"Polkadot Asset Hub Development",
-		// ID
-		"asset-hub-polkadot-dev",
-		ChainType::Local,
-		move || {
-			asset_hub_polkadot_genesis(
-				// initial collators.
-				vec![(
-					get_account_id_from_seed::<sr25519::Public>("Alice"),
-					get_collator_keys_from_seed::<AssetHubPolkadotAuraId>("Alice"),
-				)],
-				vec![
-					get_account_id_from_seed::<sr25519::Public>("Alice"),
-					get_account_id_from_seed::<sr25519::Public>("Bob"),
-					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
-				],
-				1000.into(),
-			)
-		},
-		Vec::new(),
-		None,
-		None,
-		None,
-		Some(properties),
-		Extensions { relay_chain: "polkadot-dev".into(), para_id: 1000 },
-	)
+	AssetHubPolkadotChainSpec::builder()
```

Comment:
Cleanup: change the type to:
```
pub type AssetHubPolkadotChainSpec = sc_service::GenericChainSpec<(), Extensions>;
```

All `ChainSpecs` can now have `G=()`.

Reply:
done in: https://github.com/paritytech/cumulus/pull/2936/commits/544dd82f6e83a70d29c51f91284394aa84edebee